.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: node_js
+node_js:
+  - 8
+  - node
+env:
+  global:
+  - NODE_ENV=production
+cache:
+  directories:
+    - node_modules
+install:
+  - npm --production=false install
+  - npm --production=false update
+  - npm --production=false prune
+jobs:
+  include:
+    - env: NPM_SCRIPT=lint
+      node_js: 8
```

Comment:
Sorry I missed this — you don't need this `NPM_SCRIPT=lint` unless you have `script: npm run $NPM_SCRIPT` and there's a `lint` script for the repo, which this one doesn't have yet.  So this can be removed.